Repositories.md

```diff
@@ -6,19 +6,18 @@ may be updated at any time.
 
 ## Process to Add A Repository
 
-There is a two step process for a repository to be added to the distributed
-search.
+To add a repository create a new pull request with the following two pieces
+of information:
 
-1. Provide the repository's contact information to the Helm maintainers by filling
-out the form at ABC (link to be inserted). This contact information is necessary
-when there is a need for the Helm project to contact those who manage a repository
-(e.g., it goes offline). If the repository is owned by a team or company we suggest
-using a mailing list.
-2. Create a pull request against this repository adding the repository location
-in the file at ABC (link to be inserted).
+1. In the file at `config/repo-values.yaml` add your repository with a short name
+   and the base URL to the repository.
+2. In the file `repos.yaml`, alongside this file, add contact information for
+   the repository. This contact information is necessary when there is a need for the
+   Helm project to contact those who manage a repository (e.g., it goes offline).
+   If the repository is owned by a team or company we suggest using a mailing list.
 
-Once both of these are complete the repository and its contents will be listed
-in the distributed search.
+The pull request will be reviewed. If merged it the content will be available on
```

Comment:
```suggestion
The pull request will be reviewed. If merged, the content will be available on
```

Reply:
Fixed. Not sure if the commit suggestion works with the DCO